aptos-move/move-examples/fungible_asset/stablecoin/sources/usdk.move

```diff
@@ -325,6 +335,10 @@ module stablecoin::usdk {
         assert!(!state.paused, EPAUSED);
     }
 
+    fun assert_store_not_frozen<T: key>(store: Object<T>) {
```

Comment:
@movekevin we don't need this right? Over [here](https://github.com/aptos-labs/aptos-core/blob/681a54de09d0843288fe2fb02be4de5480624200/aptos-move/framework/aptos-framework/sources/fungible_asset.move#L803):
assert!(!fa_store.frozen, error::permission_denied(ESTORE_IS_FROZEN));